app/src/main/res/values/strings_about.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
     <string name="about_activity_title">About</string>
 
-    <string name="about_description">Open source two-factor authentication for Android</string>
```

Comment:
I would like to keep this as it is now, it is kind of the tagline for andOTP

Reply:
Oh. Ok! 
Didn't know about that! 

Point noted. Would effect the chances ASAP